lib/ProductOpener/Ingredients.pm

```diff
@@ -3344,7 +3345,7 @@ sub detect_allergens_from_text($) {
 
 			# allergens between separators
 
-			#print STDERR "current text 2: $text\n";
+			print STDERR "current text 2: $text\n";
```

Comment:
We shouldn't use STDERR

Reply:
I commented it out. Can you approve the pull request?